spring-cloud-dataflow-docs/src/main/asciidoc/samples.adoc

```diff
@@ -1,8 +1,8 @@
 [[dataflow-samples]]
 = Samples
 
-Several samples have been created to help you get started implementing higher level use cases than the basic Streams and Tasks shown in the reference guide.
-The samples are part of separate https://github.com/spring-cloud/spring-cloud-dataflow-samples[repository] and have their own https://docs.spring.io/spring-cloud-dataflow-samples/docs/current/reference/html/[reference documentation].
+Several samples have been created to help you get started on implementing higher-level use cases than the basic Streams and Tasks shown in the reference guide.
+The samples are part of a separate https://github.com/spring-cloud/spring-cloud-dataflow-samples[repository] and have their own https://docs.spring.io/spring-cloud-dataflow-samples/docs/current/reference/html/[reference documentation].
```

Comment:
Let's change the doc link to htmlsingle -> https://docs.spring.io/spring-cloud-dataflow-samples/docs/current/reference/htmlsingle/ - it has been UX and easy to read in 1-go.

Reply:
I prefer to keep the list, even though there is a maintenance burden since I feel it just get overlooked if there isn't a listing in the current doc.   I even made the subsections so the get some TOC attention